assets/src/blocks/Articles/ArticlesBlock.js

```diff
@@ -36,19 +36,15 @@ export class ArticlesBlock {
           type: 'array',
           default: []
         },
-        post_types: {
-          type: 'array',
-          default: []
-        },
-        read_more_text: {
+        see_all_text: {
           type: 'string',
-          default: __('Load more', 'planet4-blocks')
+          default: __('See all articles', 'planet4-blocks')
         },
-        read_more_link: {
+        see_all_link: {
           type: 'string',
           default: ''
         },
-        button_link_new_tab: {
+        see_all_new_tab: {
```

Comment:
We could avoid changing this attribute name, if we'd want to preserve the old values. `button_link_new_tab` is still an accurate name.

If we don't want to preserve values, renaming like this makes sense to give everyone the default value..

Probably same for `read_more_text`.

Reply:
Yeah I thought about that, but I think that we do want to change the values, since it's no longer about "loading more" articles but it leads to the search page 🤔

Reply:
I guess the new tab one could eventually be kept the same, I mostly changed it for consistency

Reply:
Yeah a new name is better, it allows us to preserve the old functionality alongside the new link in the very first iteration.

Reply:
It's not the case with the current implementation though, how would that work since both buttons are at the same place in the block? 🤔